config/environments/development.rb

```diff
@@ -51,7 +51,7 @@
     Bullet.enable = true
     # Bullet.bullet_logger = true
     Bullet.rails_logger = true
-    Bullet.add_footer = true
+    # Bullet.add_footer = true
```

Comment:
Is it an intended change?

Reply:
Do people actually like having this bullet footer in the UI? 
Often it covers half of my screen, then I bring up a js console and there's no room for anything. 
I'll revert this if people prefer having it there.

Reply:
@jsimpson-cc can you revert it here, and make a new PR that uses an environment variable for it? I think the default should be on, but with an environment variable you can easily turn it off.